### ISSUE TITLE
 I can not reload anything after your example

### ISSUE THREAD

**Issue by Shleif91** [UniQHW/SH5461AS#2]
After loading your example, anything has stopped loading.
```
avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 2 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 3 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 4 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 5 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 6 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 7 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 8 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 9 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 10 of 10: not in sync: resp=0x00
An error occurred while uploading the sketch
```

**Comment by CTXz** [comment]
Would you mind telling me which IDE you are using to import the library? Further, this appears to be an error with the connection to the connection the the Arduino, not the library. Have you selected the correct Arduino model? Are you able to upload other code just fine?

Thanks

**Comment by idan424** [comment]
hey
i seem to get the same problem
is there a solution for it?
why does it do that?
`avrdude: stk500_getsync() attempt 1 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 2 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 3 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 4 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 5 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 6 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 7 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 8 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 9 of 10: not in sync: resp=0x00
avrdude: stk500_getsync() attempt 10 of 10: not in sync: resp=0x00
An error occurred while uploading the sketch`

**Comment by CTXz** [comment]
Please check if the correct board has been selected and ensure no serial communication is taking place.

**Comment by idan424** [comment]
ok so i looked for the error
the problem is you used pins 0 & 1 so as long as they're connected to something, it seems to have an error
disconnecting them solved it for me
code wise, plz change pins 0,1 to 12,13 for the future
and thanks a lot for the lib & quick response <3

**Comment by CTXz** [comment]
Ah, now I recall... Some Arduino models do not give you the option to expose or close the hw serial on pins 0 and 1, meaning the segment display was disrupting the serial communication. I'll fix the code as soon as I have access to my computer. Thanks!

**Comment by CTXz** [comment]
Fixed in 30262e9

**Comment by Guacamole-1** [comment]
Im having this error .Could you help please?

```
Example.ino:35:43: error: invalid conversion from ‘const byte* {aka const unsigned char*}’ to ‘byte* {aka unsigned char*}’ [-fpermissive]
In file included from Example.ino:29:0:
/home/guack/sketchbook/libraries/SH5461ASmaster/Display.h:49:3: note:   initializing argument 1 of ‘Display::Display(byte*, byte*, byte)’
   Display(byte digit_pins[4], byte segment_pins[7], byte dp_pin) {
   ^
Example.ino:35:43: error: invalid conversion from ‘const byte* {aka const unsigned char*}’ to ‘byte* {aka unsigned char*}’ [-fpermissive]
In file included from Example.ino:29:0:
/home/guack/sketchbook/libraries/SH5461ASmaster/Display.h:49:3: note:   initializing argument 2 of ‘Display::Display(byte*, byte*, byte)’
   Display(byte digit_pins[4], byte segment_pins[7], byte dp_pin) {
   ^
```
